src/core/model.js

```diff
@@ -116,6 +117,16 @@ export default class Model extends Object3D {
 
     this.onBeforeRender = onBeforeRender;
     this.onAfterRender = onAfterRender;
+
+    this.timeElapsedQuery = undefined;
```

Comment:
A bit too much code to add here, would be best to have a separate method/util to handle this.